lte/protos/subscriberdb.proto

```diff
@@ -210,6 +209,22 @@ service SubscriberDB {
   // List the subscribers in the store.
   //
   rpc ListSubscribers (magma.orc8r.Void) returns (SubscriberIDSet) {}
+
```

Comment:
Can you move it to subscriberdb changes.

Reply:
Done